provelo_custom/views/hr_kanban_views_employees.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0"?>
+<odoo>
+    <record id="hr_kanban_view_employee_extend" model="ir.ui.view">
```

Comment:
[OCA convention](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#253inherited-xml) is to use he same id for the extending view (hr_kanban_view_employees)

Reply:
Done !